### DISCUSSION TITLE
How to integrate remix-i18next after switching from renderToString to renderToPipeableStream?

### DISCUSSION THREAD

**Discussion by hiql** [remix-run/remix#3785] [Q&A]
After the stack changed the server render method from  renderToString to renderToPipeableStream, I think it‘s a big challenge when you integrate 3rd react libs(remix-i18next、chakra ui...).
Does anyone know how to make it work?

**Comment by MichaelDeBoey** [comment]
I think it's best to ask this questions on https://github.com/sergiodxa/remix-i18next, where @sergiodxa will answer them

**Comment by sergiodxa** [comment]
What error do you get? Is it with remix-i18next or with react-i18next?

**Comment by hiql** [comment]
Apologies, guys
The lib has no problem.

Did you notice that the latest blue stack has changed a lot? Now it's based on react 18 streaming, especially in the file "entry.server.tsx".
I am looking for some example code.I just have no idea how to migrate to the latest blue stack?
Will you update the example code?

**Comment by tombohub** [comment]
did you figure out how to add chakra?
